pkg/operator/backupcontroller/backupcontroller.go

```diff
@@ -65,8 +67,11 @@ func NewBackupController(livenessChecker *health.MultiAlivenessChecker,
 	syncer := health.NewDefaultCheckingSyncWrapper(c.sync)
 	livenessChecker.Add("BackupController", syncer)
 
-	return factory.New().ResyncEvery(1*time.Minute).
-		WithSync(syncer.Sync).ToController("BackupController", eventRecorder.WithComponentSuffix("backup-controller"))
+	return factory.New().
+		ResyncEvery(1*time.Minute).
+		WithInformers(backupInformer.Informer()).
```

Comment:
This controller also needs to watch Jobs so that it can reconcile when they fail or succeed right?
```suggestion
		WithInformers(backupInformer.Informer(),
		kubeInformers.InformersFor(operatorclient.TargetNamespace).Batch().V1().Jobs().Informer()
		).
```
where kubeInformers is:
https://github.com/openshift/cluster-etcd-operator/blob/e3d686d9e7125514a9045ecdb92263a50ed961ba/pkg/operator/starter.go#L136

Reply:
thanks and done, I'll do a quick test run still

Reply:
glad I did, needed to start its informer separately as well. All fixed now.

Reply:
hmm no, CI still complains about it restarting...

Reply:
ah dang, that makes sense:

```
W0726 12:54:27.942842       1 reflector.go:533] k8s.io/client-go@v0.27.4/tools/cache/reflector.go:231: failed to list *v1alpha1.EtcdBackup: the server could not find the requested resource (get etcdbackups.operator.openshift.io)
E0726 12:54:27.942864       1 reflector.go:148] k8s.io/client-go@v0.27.4/tools/cache/reflector.go:231: Failed to watch *v1alpha1.EtcdBackup: failed to list *v1alpha1.EtcdBackup: the server could not find the requested resource (get etcdbackups.operator.openshift.io)
W0726 12:54:27.942894       1 reflector.go:533] k8s.io/client-go@v0.27.4/tools/cache/reflector.go:231: failed to list *v1alpha1.Backup: the server could not find the requested resource (get backups.config.openshift.io)
E0726 12:54:27.942902       1 reflector.go:148] k8s.io/client-go@v0.27.4/tools/cache/reflector.go:231: Failed to watch *v1alpha1.Backup: failed to list *v1alpha1.Backup: the server could not find the requested resource (get backups.config.openshift.io)
```

especially when the cluster isn't running techpreview enabled